lighthouse-core/test/audits/byte-efficiency/total-byte-weight-test.js

```diff
@@ -90,4 +90,25 @@ describe('Total byte weight audit', () => {
       assert.strictEqual(result.score, 0);
     });
   });
+
+  it('ignores non-network requests', async () => {
+    const artifacts = generateArtifacts([
+      {url: 'https://example.com/file.js', transferSize: 10_000},
+      {url: 'data:image/jpeg;base64,', protocol: 'data', transferSize: 100_000},
+      {url: 'blob:1234', protocol: 'blob', transferSize: 100_000},
+    ]);
+
+    const result = await TotalByteWeight.audit(artifacts, {options, computedCache: new Map()});
+    expect(result.numericValue).toEqual(10_000);
+  });
+
+  it('does not throw on requests with NaN transferSize', async () => {
```

Comment:
nit: I was expecting something like
```suggestion
  it('ignores requests with falsy transfer size', async () => {
```